### PR TITLE
JTeUY1ZZ Setup React project

### PR DESCRIPTION
**Pull request by KundwaBruno**
### What does this PR do?
This PR sets up the initial react project
### Description of Task to be completed?

- The project is configured from scratch using Wepack and Babel
- Setup react router and create a dummy route `"/"` for the landing page
- Added a readMe file which includes installation and run commands
- SCSS has been added as our choice of CSS framework

### How should this be manually tested?

- Run `npm install` to install all dependecies.
- Run `npm run dev` to start your application.

### Relevant Trello card

ID: JTeUY1ZZ

### CODE REVIEW

**Review by mansurissa — APPROVED** [review]
Nice Job

**Review by samuel776 — APPROVED** [review]

**Review by SIBONIYO — APPROVED** [review]
Great Job Bruno and the team!

**Review by raymond42 — CHANGES_REQUESTED** [review]
attend to the comment and revert to me

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...la/Desktop/phantom-frontend/incredible-phantom-frontend/src/componets/welcomeMessage.js":"4"},{"size":195,"mtime":1609229359713,"results":"5","hashOfConfig":"6"},{"size":291,"mtime":1609229359714,"results":"7","hashOfConfig":"6"},{"size":177,"mtime":1609229359713,"results":"8","hashOfConfig":"6"},{"size":342,"mtime":1609229359713,"results":"9","hashOfConfig":"6"},{"filePath":"10","messages":"11","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1axnbbs",{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/andela/Desktop/phantom-frontend/incredible-phantom-frontend/src/index.js",[],"/Users/andela/Desktop/phantom-frontend/incredible-phantom-frontend/src/router/App.js",[],"/Users/andela/Desktop/phantom-frontend/incredible-phantom-frontend/src/pages/landingPage.js",[],"/Users/andela/Desktop/phantom-frontend/incredible-phantom-frontend/src/componets/welcomeMessage.js",[]]
```
Comment (raymond42) [inline .eslintcache]: can we stop tracking this file? you can put it in `.gitignore` file

**Review by raymond42 — APPROVED** [review]
LGTM

**Pull request merged**